app/assets/javascripts/discourse/app/models/user-draft.js

```diff
@@ -33,10 +33,10 @@ export default class UserDraft extends RestModel {
 
   @discourseComputed("draft_key")
   draftType(draftKey) {
-    switch (draftKey) {
-      case NEW_TOPIC_KEY:
+    switch (true) {
```

Comment:
Did you mean to change this to `true`? If so, maybe remove the whole `switch` instead?